### ISSUE TITLE
Controllers (and workers) need an auto-shutdown feature

### ISSUE THREAD

**Issue by briandrawert** [Molns/molns#13]

**Comment by ahellander** [comment]
I dont think this is possible yet in os, but jt ls more important in aws anyway.

Skickat från min Samsung Mobil.

-------- Originalmeddelande --------
Från: Brian Drawert
Datum:01-04-2015 18:03 (GMT+01:00)
Till: Molns/molns
Rubrik: [molns] Controllers (and workers) need an auto-shutdown feature (#13)
